web/src/components/Firebase/Firebase.js

```diff
@@ -17,6 +20,75 @@ const firebaseClient = ((config) => {
   return firebase
 })(firebaseClientConfig)
 
+const FirebaseUserTools = () => {
+  const [email, setEmail] = useState('')
+  const [password, setPassword] = useState('')
+  const { logIn, logOut, signUp, isAuthenticated } = useAuth()
+
+  return (
+    <div>
+      <h2>firebase</h2>
+      {isAuthenticated ? 'Authenticated' : 'Not Authenticated'}{' '}
+      <h3>Google Login (default)</h3>
+      <LogInOutButtons />
+      {/* Other providers require setup in Firebase account */}
+      <h3>Apple Login</h3>
```

Comment:
In the interest of decluttering the page, could you perhaps have a dropdown where you pick the provider. Or a vertical list of toggle buttons. And then just one set of signup/login buttons

Reply:
Done